core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -1548,8 +1548,11 @@ private void updateUrlProcessor() {
   }
 
   private void updateNightMode() {
-    painter.update(getCurrentWebView(), kiwixWebView -> kiwixWebView.getUrl() == null
-      || !kiwixWebView.getUrl().equals(HOME_URL), videoView);
+    painter.update(getCurrentWebView(), this::shouldActivateNightMode, videoView);
+  }
+
+  private boolean shouldActivateNightMode(KiwixWebView kiwixWebView) {
+    return kiwixWebView!=null && !HOME_URL.equals(kiwixWebView.getUrl());
```

Comment:
Formatting. Space should be added before & after `!=`.

Reply:
Thanks for the catch, stupid java. Fixed